src/EnergyPlus/SizingManager.cc

```diff
@@ -816,7 +819,7 @@ void ManageSizing(EnergyPlusData &state)
                                 FinalSysSizing(AirLoopNum).DesCoolVolFlow,
                                 FinalSysSizing(AirLoopNum).CoolDesDay,
                                 coolPeakDDDate,
-                                SysSizPeakDDNum(AirLoopNum).TimeStepAtHeatPk(coolPeakDD));
+                                timeStepIndexAtPeakCoolLoad);
```

Comment:
This is the offending code that needed to change. It should point to `TimeStepAtTotCoolPk(coolPeakDD)`, instead of `TimeStepAtHeatPk(coolPeakDD)`.

Reply:
This makes good sense.

Reply:
Yes, I agree with this change.